utils/src/ooinstall/openshift_ansible.py

```diff
@@ -131,6 +131,9 @@ def write_host(host, inventory, schedulable=None):
     elif not schedulable:
         facts += ' openshift_schedulable=False'
 
+    if host.is_infra_node:
+        facts += ' region=Infra'
```

Comment:
I almost wonder if handling region in a better way would be a good idea... 
If allowing arbitrary config, the user could be specifying the regions. It'd also be nice to eventually support the concept of regions/zones in the guided config as well.